src/ko.json

```diff
@@ -1398,13 +1398,14 @@
             "activate": "활성화",
             "assets": "자산",
             "assetsBalance": "자산 잔액",
-            "buy": "토큰 구입",
+            "buy": "Buy",
```

Comment:
@kkkhesha could you give me a translation, pls?)

Reply:
@vetkamak done)